### PR TITLE
Backport/parsing responses for lti preview

### PR DESCRIPTION
**Pull request by bugalot**
This PR is a backport of https://github.com/oat-sa/extension-lti-outcomeui/pull/57

### CODE REVIEW

File: controller/ItemResultPreviewer.php [hunk 1]
```diff
@@ -149,10 +149,8 @@ protected function getItemResultVariables(core_kernel_classes_Resource $delivery
     {
         $variable = $this->getItemVariable($delivery, $resultIdentifier, $itemDefinition);
         if (isset($variable['uri'])) {
-            $responses = ResponseVariableFormatter::formatStructuredVariablesToItemState([$variable]);
-            if (isset($responses[$variable['uri']])) {
-                return $responses[$variable['uri']];
-            }
+            $responses = $this->getResultVariableStructureHandler()->format([$variable]);
```
Comment (tikhanovichA) [inline controller/ItemResultPreviewer.php:152]: are we sure that it will work?

**Review by btamas — APPROVED** [review]

**Review by btamas — CHANGES_REQUESTED** [review]
```

Fatal error: Uncaught Error: Call to undefined method oat\ltiOutcomeUi\controller\ItemResultPreviewer::getResultVariableStructureHandler() in /var/www/html/ltiOutcomeUi/controller/ItemResultPreviewer.php:152 Stack trace: #0 /var/www/html/ltiOutcomeUi/controller/ItemResultPreviewer.php(106): oat\ltiOutcomeUi\controller\ItemResultPreviewer->getItemResultVariables(Object(core_kernel_classes_Resource), 'https://tao.doc...', 'I_21') #1 [internal function]: oat\ltiOutcomeUi\controller\ItemResultPreviewer->preview() #2 /var/www/html/tao/models/classes/routing/ActionEnforcer.php(239): call_user_func_array(Array, Array) #3 /var/www/html/tao/models/classes/routing/ActionEnforcer.php(192): oat\tao\model\routing\ActionEnforcer->resolve(Object(GuzzleHttp\Psr7\ServerRequest)) #4 /var/www/html/tao/models/classes/routing/ActionEnforcer.php(169): oat\tao\model\routing\ActionEnforcer->execute() #5 /var/www/html/tao/models/classes/routing/TaoFrontController.php(101): oat\tao\model\routing\ActionEnforcer->__invoke(Object(GuzzleHttp\Psr7\Serv in /var/www/html/ltiOutcomeUi/controller/ItemResultPreviewer.php on line 152
```

File: controller/ItemResultPreviewer.php (hunk 1 above)
Comment (btamas) [reply controller/ItemResultPreviewer.php:152]: It doesn't work

Comment (tikhanovichA) [reply controller/ItemResultPreviewer.php:152]: yes we should have only `return current($responses);`

**Comment by bugalot** [comment]
Because an old branch of an aborted released had fun with us

**Pull request closed without merging**